site/Gemfile

```diff
@@ -15,8 +15,8 @@
 #
 source 'https://rubygems.org'
 gem 'jekyll', '~>4'
-gem "webrick", "~> 1.7"
-gem "nokogiri", "~> 1.14"
+gem "webrick", "~> 1.9"
```

Comment:
Hi @hughpearse , your commit info should be like "[CALCITE-XX] Jira ticket summary". See https://calcite.apache.org/develop/#contributing.
By the way, how can we get the scan result after your fix, in 1.39.0 release?